packages/babel-preset-anansi/index.js

```diff
@@ -137,32 +138,26 @@ function buildPreset(api, options = {}) {
         targets: {
           node: options.nodeTarget || 'current',
         },
+        bugfixes: true,
         modules,
         // maximum compatibility since we don't care about bundle size
         useBuiltIns: false,
       },
     ]);
   } else {
-    // TODO: enable this in more cases based on browserslist
-    if (options.targets && options.targets.esmodules === true) {
-      preset.presets.unshift([
-        require('@babel/preset-modules'),
```

Comment:
this is now obsoleted by 'bugfixes: true' option, which applies more generally as well